guide/docs/popular-topics/permissions.mdx

```diff
@@ -1,8 +1,382 @@
+superset = disnake.Permissions(add_reactions=True, manage_guild=True, send_messages=True)
+subset = disnake.Permissions(add_reactions=True)
+different = disnake.Permissions(use_voice_activation=True)
+```
+
+### Basic
+
+These are operations that usually require no explanation. In any way, comments will guide you through.
+
+```python
+# Check that two permission object have same permissions set to the same value.
+assert base == same
+# Check that one (base) permission object has at least all of the granted permissions
+# of the second one granted too.
+assert base >= same
+# Check that one (base) permission object has all of the granted permissions
+# of the second one granted too, plus at least one extra.
+assert base > subset
+# Check that one (base) permission object has at maximum all of the granted permissions
+# of the second one granted.
+assert base <= same
+# Check that one (base) permission object has at maximum of the granted permissions
+# of the second one granted, minus one or more.
+assert base < superset
+# Check that one (base) permission object has at least one permission granted/revoked
+# that is revoked/granted in the second one.
+assert base != different
+```
+
+### Advanced
+
+If you're familar with bit operators in Python, then you'll feel right at home. Otherwise, read
+the comments to understand what does what.
+
+```python
+# Creates a new Permissions object with all granted permissions from both base and different.
+assert subset | different == disnake.Permissions(add_reactions=True, use_voice_activation=True)
+# Creates a new Permissions object with only permissions granted on both base and different.
+assert base & subset == disnake.Permissions(add_reactions=True)
+# Creates a new Permissions object with only permissions granted on one of base or different,
+# but not both.
+assert base ^ different
+
+# Notably, all of operators above support in-place updates using the `OP=` syntax, like so:
+# `base &= subset`.
+
+# Creates a new Permissions object with all permissions from subset inverted.
+assert not (~subset).add_reactions and (~subset).manage_guild
+# Creates a new Permissions object with all permissions except add_reactions granted.
+assert not (~disnake.Permissions.add_reactions).add_reactions
+# Creates a new Permissions object with all specified permissions granted.
+assert disnake.Permissions.add_reactions | disnake.Permissions.manage_guild == base
+assert subset | disnake.Permissions.manage_guild == base
+```
+
+### Additional
+
+There's also a few additional operations supported that do not involve operators.
+
+```python
+# Returns the permissions's hash. This currently returns it's value, but that may change
+# in the future.
+assert hash(base) == base.value == 96
+
+# Permissions object implements `__iter__()`, and thus can be used in for loops.
+for perm, value in base:
+    truthy = []
+
+    if value:
+        truthy.append(perm)
+
+    assert truthy == ["add_reactions", "manage_guild"]
```

Comment:
imo this might be worth its own section, it's often overlooked and is supported by both `Permissions` and `PermissionOverwrite`

Reply:
I'm not quite sure how to fill such section (there isn't anything much to talk about or to explain). By the way fixed the example (b84f27e).

Reply:
Fair point, it would be difficult to move this into a separate section since the functionality is pretty straightforward.
Added another comment here, but overall this looks great: https://github.com/DisnakeDev/guide/pull/58#discussion_r1258096528